src/app/pages/common/ix-forms/components/ix-jira-oauth/ix-jira-oauth.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="full-width container has-tooltip">
+
+  <div *ngIf="label" class="label-container">
+    <label class="label">
+      {{label}}
+      <span *ngIf="required" class="required">*</span>
+    </label>
+    <tooltip *ngIf="tooltip" [header]="label" class="tooltip" [message]="tooltip"></tooltip>
+  </div>
+
+  <div class="outlined input-container" [class.has-reset-button]="shouldShowResetInput()">
+    <input
```

Comment:
Help text on Login to Jira is good, but we don't really need to show token to the user - there is nothing he can do with it.